SalesforceCommerceCloud/pwa-kit
# Consistent Logs - Add logger to pwa-kit-create-app (@W-15832306@)

# Description



Follow up PR to https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1822 adding the `logger` to pwa-kit-create-app

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Added logger to pwa-kit-create-app

# How to Test-Drive This PR

- Generate a project:
```
node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir ~/generated-project-demo2
```
- Start the dev server on the generated project and verify the logs include the namespace: `npm start`
# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
